boards/arm/mec15xxevb_assy6853/doc/index.rst

```diff
@@ -0,0 +1,281 @@
+PVT SPI, SHD SPI and LED0-2 respectively.
+
++---------+--------+-----------+----------+---------+
+| JP68    | JP9    | JP38      | JP98     | JP41    |
+| (UART0) | (JTAG) | (PVT SPI) | (SHD SPI)| (LED0-2)|
++=========+========+===========+==========+=========+
+| 1-2     | 2-3    | 2-3       | 2-3      | 1-2     |
++---------+--------+-----------+----------+---------+
+| 4-5     | 5-6    | 5-6       | 5-6      | 3-4     |
++---------+--------+-----------+----------+---------+
+| 8-9     | 8-9    | 8-9       | 8-9      | 5-6     |
++---------+--------+-----------+----------+---------+
+|         | 11-12  | 11-12     | 11-12    |         |
++---------+--------+-----------+----------+---------+
+|         |        | 14-15     | 14-15    |         |
++---------+--------+-----------+----------+---------+
+|         |        | 17-18     | 20-21    |         |
++---------+--------+-----------+----------+---------+
+
+Jumper settings for MEC1501 144WFBGA Socket DC Assy 6883 Rev B1p0
+=================================================================
+
+The jumper configuration explained above covers the base board. The ASSY
+6883 MEC1501 CPU board provides capability for an optional, external 32KHz
+clock source. The card includes a 32KHz crystal oscillator. The card can
+also be configured to use an external 50% duty cycle 32KHz source on the
+XTAL2/32KHZ_IN pin. Note, firmware must set the MEC15xx clock enable
+register to select the external source matching the jumper settings. If
+using the MEC15xx internal silicon oscillator then the 32K jumper settings
+are don't cares. JP1 is for scoping test clock outputs. Please refer to
+the schematic in reference section below.
+
+Parallel 32KHz crystal configuration
+------------------------------------
++-------+-------+
+| JP2   | JP3   |
++=======+=======+
+| 1-2   | 2-3   |
++-------+-------+
+
+External 32KHz 50% duty cycle configuration
+-------------------------------------------
++-------+-------+
+| JP2   | JP3   |
++=======+=======+
+| NC    | 1-2   |
++-------+-------+
+
+
+Jumper settings for MEC1503 144WFBGA Socket DC Assy 6856 Rev B1p0
+=================================================================
+
+The MEC1503 ASSY 6856 CPU card does not include an onboard external
+32K crystal or oscillator. The one jumper block JP1 is for scoping
+test clock outputs not for configuration. Please refer to schematic
+in reference section below.
+
+Programming and Debugging
+*************************
+
+This board comes with a Cortex ETM port which facilitates tracing and debugging
+using a single physical connection.  In addition, it comes with sockets for
+JTAG only sessions.
+
+Flashing
+========
+
+#. Connect the SPI Dongle ASSY 6791 to J36 (SPI dongle) in order to flash and
+   boot from SHD SPI NOR. Then proceed to flash using Dediprog SF100 or a
+   similar tool for flashing SPI chips.
+   note:: Remember that SPI MISO/MOSI are swapped on dediprog headers!
```

Comment:
Because of the double colons, I suspect you wanted this to appear as a "note", so add a blank line before this and change to:
```
.. note:: Remember that SPI MISO/MOSI are swapped on dediprog headers!
```

Reply:
Thanks! Pushed update with your recommended change.